### ISSUE TITLE
Bash Autocomplete Depends on bash-completion Library

### ISSUE THREAD

**Issue by sshaw** [ggreer/the_silver_searcher#1067]
[It uses `_get_cword`](https://github.com/ggreer/the_silver_searcher/blob/da79caccf89f2a84287a2a4e99bdcbc48e924771/ag.bashcomp.sh#L6), which is part of the [bash-completion library](https://github.com/scop/bash-completion).

**Comment by jschpp** [comment]
I'm not seeing the problem here. If you want to use bash autocomplete shouldn't you have the bash-completion library installed anyhow?

**Comment by sshaw** [comment]
>  If you want to use bash autocomplete shouldn't you have the bash-completion library installed anyhow?

No, an external library is not necessary. Bash has [builtin commands for this](https://www.gnu.org/software/bash/manual/html_node/Programmable-Completion-Builtins.html).

**Comment by josephholsten** [comment]
@sshaw I'd love to see how much code is required to remove the dependency on bash-completion. Care to throw together a PR for us?
